src/NuGet.Clients/NuGet.VisualStudio.Implementation/NuGet.VisualStudio.Implementation.csproj

```diff
@@ -6,6 +6,7 @@
     <Description>Implementation of the NuGet.VisualStudio extensibility APIs.</Description>
     <Guid>9623cf30-192c-4864-b419-29649169ae30</Guid>
     <ImportedFromTypeLib>NuGet.VisualStudio.Implementation</ImportedFromTypeLib>
```

Comment:
I'm confused as to why we need all these. 
Aren't the projects scoped down differently?

I think we can probably solve this in a different file where it affects all projects in the src/NuGet.Clients subfolder.

Reply:
I can probably make that change myself, if you prefer not add more complications in this PR.

Reply:
The issue is that these projects don't build properly in the VMR in non-source-only modes. I think typically in non-Windows cases. The goal is to exclude as little as possible from the default repo build in VMR modes. This is the only case of this. The rest of the Exclude* are just changing the source-only exclusion property to match the new name.

Reply:
I noticed the prop change, I was more thinking that we shouldn't have to mark each individual project, since source build only cares about nuget.core projects with the exception of NuGet.PackageManagement, so there should be easier ways to do this (example, directory.build.props).

Reply:
Generally, we've tried to avoid proactively trimming down the set of projects to build, especially in non-source-only modes. My view is that the repo should mostly work 'stock', and we should avoid removing extras if not necessary.

If you want to stop building more projects, that's fine, but I think it's probably good that we only exclude 3 projects.